src/Umbraco.Web.UI/umbraco/config/lang/da.xml

```diff
@@ -1254,7 +1254,7 @@ Mange hilsner fra Umbraco robotten
     <key alias="addRowConfigurationDetail">Juster rækken ved at indstille cellebredder og tilføje yderligere celler</key>
 
     <key alias="columns">Kolonner</key>
-    <key alias="columnsDetails">Det totale antaller kolonner i gitteret</key>
+    <key alias="columnsDetails">Det totale antaller kolonner i dit grid</key>

```

Comment:
@BatJan thanks a lot for creating a PR 👍 

I agree that we should just use **grid** instead of **gitter**. The other changes look fine as well.

But while you're at it, could you change **antaller** to just **antal** as well (unless you don't agree)? Then you'll get an extra high five from me 👏

Reply:
Hey @abjerner - Well yes of course! Good catch did not notice that one myself :)